_plugins/version.rb

```diff
@@ -0,0 +1,52 @@
+# Aggregate the En lesson versions together
+Jekyll::Hooks.register :site, :post_read do |site|
```

Comment:
If we use a `:site` `:pre_render` hook we can create a page/report for outstanding translation work.